examples/ButtonMouseControl/ButtonMouseControl.ino

```diff
@@ -0,0 +1,75 @@
+/*
+  ButtonMouseControl
+
+  For native USB boards (e.g., Leonardo, Micro, MKR, Nano 33 IoT, Zero, Due) only.
+
+  Controls the mouse from five pushbuttons.
+
+  Hardware:
+  - five pushbuttons attached to D2, D3, D4, D5, D6
+
+  The mouse movement is always relative. This sketch reads four pushbuttons,
+  and uses them to set the movement of the mouse.
+
+  WARNING: When you use the Mouse.move() command, the Arduino takes over your
+  mouse! Make sure you have control before you use the mouse commands.
```

Comment:
```suggestion
  mouse! Make sure you have control before you use the mouse commands.

  This example code is in the public domain.

  http://www.arduino.cc/en/Tutorial/ButtonMouseControl
```
I think it's important to retain the license information from https://www.arduino.cc/en/Tutorial/ButtonMouseControl and also the link back to the tutorial.